kratos/utilities/binbased_nodes_in_element_locator.h

```diff
@@ -121,11 +121,19 @@ class BinBasedNodesInElementLocator
     ///function to find all teh nodes of a fixed mesh contained in the elements of a moving mesh
     /**
     *	It is called each time a projection from a moving to a fixed mesh is performed using the function of @see binbased_projection.h
+    * @param pelement The pointer to the element to be studied
+    * @param positions The indexes of the relative positions
+    * @param Nmat Shape functions
+    * @param max_results Maximum number of results to be searched
+    * @param work_results The point iterator with the points to be studied
+    * @param work_distances The iterator containing the relative distances
+    * @param work_point The node of study
+    * @return counter: The number of iterations needed
     */
     unsigned int FindNodesInElement(Element::Pointer& pelement,
                                     boost::numeric::ublas::vector<int>& positions,
                                     Matrix& Nmat,
-                                    const unsigned int& max_results,
+                                    const unsigned int max_results,
```

Comment:
You may not change an argument type of a class in a PR for doxygen

Reply:
Sorry, not again